src/ContactComponent.php

```diff
@@ -162,19 +162,15 @@ function wf_crm_contact_search($node, $component, $params, $contacts, $str = NUL
         }
       }
       // If we get exactly $limit results, there are probably more - warn that the list is truncated
-      // TODO Error reporting is still Drupal7-ish
       if (wf_crm_aval($result, 'count') >= $limit) {
-        watchdog(
-          'webform_civicrm',
-          'Maximum contacts exceeded, list truncated on the webform "@title". The webform_civicrm "@field" field cannot display more than !limit contacts because it is a select list. Recommend switching to autocomplete widget in component settings.',
-          array('!limit' => $limit, '@field' => $component['name'], '@title' => $node->title),
-          WATCHDOG_WARNING,
-          l(t('Edit component'), "node/{$node->nid}/webform/components/{$component['cid']}")
+        \Drupal::logger('webform_civicrm')->warning(
+          'Maximum contacts exceeded, list truncated on the webform "@title". The webform_civicrm "@field" field cannot display more than @limit contacts because it is a select list. Recommend switching to autocomplete widget in component settings.',
+          array('@limit' => $limit, '@field' => $component['#title'], '@title' => $node->label()),
         );
-        if (wf_crm_admin_access($node) && node_is_page($node)) {
-          \Drupal::messenger()->addWarning('<strong>' . t('Maximum contacts exceeded, list truncated.') .'</strong><br>' .
-          t('The field "@field" cannot show more than !limit contacts because it is a select list. Recommend switching to autocomplete widget in <a !link>component settings</a>.',
-            array('!limit' => $limit, '@field' => $component['name'], '!link' => 'href="' . url("node/{$node->nid}/webform/components/{$component['cid']}") . '"')));
+        if ($node->access('update') && \Drupal::currentUser()->hasPermission('access CiviCRM')) {
+          $warning_message = \Drupal\Core\Render\Markup::create('<strong>' . t('Maximum contacts exceeded, list truncated.') .'</strong><br>' .
```

Comment:
I don't _think_ `Markup::create` is required for `<strong>`.

Reply:
Hey Matt - if I don't use `Markup::create` the `<strong>` and `<br>` are rendered literally.  So I'm not sure if I should leave in `Markup::create` or you're suggesting there's a more lightweight `MarkupInterface` I should be using instead.

Reply:
Oh! Nevermind then :)

Reply:
Ok resolved then! 👍